services/rule_engine/src/main/resources/config/signals.yaml

```diff
@@ -13,3 +13,8 @@ com.mindsmiths.armory.event.UserDisconnected:
     connectionName: armory
     connectionField: connectionId
     agentType: agents.Moli
+com.mindsmiths.infobipAdapter.models.InfobipReceivedMessage:
```

Comment:
We might not want this. Currently, this will create a new agent when a random person contacts us over Whatsapp. We need to ask product about this.